src/BridgeMaker.js

```diff
@@ -7,7 +7,18 @@ const BridgeMaker = {
    * @param {function(): number} generateRandomNumber 무작위 값을 생성해주는 함수
    * @return {string[]} 입력받은 길이에 해당하는 다리 모양. 위 칸이면 U, 아래 칸이면 D로 표현해야 한다.
    */
-  makeBridge(size, generateRandomNumber) {},
+  makeBridge(size, generateRandomNumber) {
+    const answerBridge = [];
+
+    for (let i = 0; i < size; i++) {
+      const randomNumber = generateRandomNumber();
+
+      if (parseInt(randomNumber) === 0) answerBridge.push("D");
+      if (parseInt(randomNumber) === 1) answerBridge.push("U");
+    }
```

Comment:
```suggestion
    for (let i = 0; i < size; i++) {
      answerBridge.push( generateRandomNumber() ? "U" : "D");
    }
```
코드 상에서 0은 false로, 1은 true로 인식한다고 해요.
이 부분을 활용해서 if문 대신 삼항연산자를 활용해서 이런 방식으로 중복코드를 줄여보는건 어떨까요 ?:)

Reply:
"U"와 "D"는 의미를 가지고 있는 값이니 상수 처리를 해서 가지고 와서 사용하는게 좋을 것 같습니다 !

Reply:
상수 처리를 다 했다고 생각했는데 남는 부분이 있었네요! 코드를 완성하고 상수 처리를 하지 않고 코드를 작성하면서 상수 처리를 해야겠어요!! 감사합니당